server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java

```diff
@@ -1763,8 +1763,12 @@ private void parseInsertValuesSpec(InsertValuesSpecContext ctx, InsertOperator i
       long timestamp;
       if (insertMultiValues.get(i).dateFormat() != null) {
         timestamp = parseTimeFormat(insertMultiValues.get(i).dateFormat().getText());
-      } else {
+      } else if (insertMultiValues.get(i).INT() != null) {
         timestamp = Long.parseLong(insertMultiValues.get(i).INT().getText());
+      } else if (insertMultiValues.size() != 1) {
+        throw new SQLParserException("need timestamps when insert multi rows");
+      } else {
+        timestamp = parseTimeFormat("now()");
```

Comment:
```suggestion
        timestamp = parseTimeFormat(SQLConstant.NOW_FUNC);
```

Reply:
@HTHou Thank you for your good suggestion